elide-integration-tests/src/test/java/com/yahoo/elide/async/integration/tests/framework/AsyncIntegrationTestApplicationResourceConfig.java

```diff
@@ -88,15 +94,15 @@ protected void configure() {
                         .withSubqueryFilterDialect(multipleFilterStrategy)
                         .withEntityDictionary(dictionary)
                         .withISO8601Dates("yyyy-MM-dd'T'HH:mm'Z'", Calendar.getInstance().getTimeZone())
+                        .withExportApiPath("/export")
                         .build());
                 bind(elide).to(Elide.class).named("elide");
 
                 AsyncAPIDAO asyncAPIDao = new DefaultAsyncAPIDAO(elide.getElideSettings(), elide.getDataStore());
                 bind(asyncAPIDao).to(AsyncAPIDAO.class);
 
-                // TODO Pass resultStorageEngine to TableExportHook
-                ResultStorageEngine resultStorageEngine =
-                        new FileResultStorageEngine(System.getProperty("java.io.tmpDir"));
+                // Create ResultStorageEngine using /tmp
+                ResultStorageEngine resultStorageEngine = new FileResultStorageEngine("/tmp");
```

Comment:
We should use Java libraries here that manage tmp files (rather than using /tmp).  This will also likely fail on windows systems.